engine/lib/phx/src/system/profiler.rs

```diff
@@ -16,8 +16,8 @@ pub type Signal = i32;
 pub struct Scope {
     pub name: *const libc::c_char,
     pub last: TimeStamp,
-    pub frame: TimeStamp,
-    pub total: TimeStamp,
+    pub frame: u64,
```

Comment:
I think if we're storing `u64`'s here, we're losing some context as to what the numbers represent. Can we not continue to store `TimeStamp`'s here?

Reply:
frame and total are not really TimeStamps but duration. At least it comes from their usage. Correct me if I'm wrong.
Also original type for them was u64 (aliased to TimeStamp) so I just kept it here explicitly.

Reply:
We can even use std::time::Duration directly but in both cases it will require additional changes in several places of the engine.
I'd prefer to do it later